### ISSUE TITLE
npm install fails on Ubuntu on Windows

### ISSUE THREAD

**Issue by fsih** [LLK/scratch-gui#2532]
### Expected Behavior
npm install works

### Actual Behavior
npm install fails. The error is of the form
```
npm ERR! path /mnt/c/Users/DD/Git/scratch-gui/node_modules/.staging/js-md5-bebc6219/tests/worker.html
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall utime
npm ERR! enoent ENOENT: no such file or directory, utime '/mnt/c/Users/DD/Git/scratch-gui/node_modules/.staging/js-md5-bebc6219/tests/worker.html'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent
```
It has failed on a bunch of different files for me, but always somewhere in .staging
```
chown '/mnt/c/Users/DD/Git/scratch-gui/node_modules/.staging/minimalistic-crypto-utils-d47758ba/.npmignore'
utime '/mnt/c/Users/DD/Git/scratch-gui/node_modules/.staging/js-md5-bebc6219/tests/worker.html'
utime '/mnt/c/Users/DD/Git/scratch-gui/node_modules/.staging/rxjs-d97613be/_esm5/internal/observable/fromPromise.js.map'
utime '/mnt/c/Users/DD/Git/scratch-gui/node_modules/.staging/enzyme-2ceb6b36/ReactWrapper.js'

```
### Steps to Reproduce
Run npm install on scratch-gui in ubuntu on windows. The gui repo is on the windows mnt.
### Operating System and Browser
Windows

**Comment by fsih** [comment]
It may have to do with the file name beginning with a . ?

**Comment by fsih** [comment]
Fixed by updating to npm 5.6.0 and nodejs 8.11.3
